src/NCronJob/Scheduler/JobQueueManager.cs

```diff
@@ -65,8 +65,8 @@ public void RemoveQueue(string queueName)
 
     public IEnumerable<string> GetAllJobQueueNames() => jobQueues.Keys;
 
-    public SemaphoreSlim GetOrAddSemaphore(string jobType, int concurrencyLimit) =>
```

Comment:
This is only ever called by `JobWorker.WorkerAsync()`.
The passed in parameter from that method is `queueName`. This renaming makes it more uniform and easier to reason about in the context of a JobQueueManager.

Reply:
True that. Well, in most cases the jobType is the name of the queue. as we have, ideally, one queue per jobtype. But I like this more!